dev/CommonStyles/FlyoutPresenter_themeresources.xaml

```diff
@@ -52,19 +54,23 @@
                         BorderThickness="{TemplateBinding BorderThickness}"
                         contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                         contract7Present:BackgroundSizing="OuterBorderEdge">
```

Comment:
Do the same thing we did for tooltip.

Make this a TemplateBinding: contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"

With setter above: <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />